src/constants/dummyData.js

```diff
@@ -1,10 +1,152 @@
-// candidates
-// employees
-// jobs
-// notifications
-// onBoarding
-// organizations
-// project onbording
-// birthdays
-// work anniversaries
-// holidays
+export const dummyData = {
+  employees: [
+    {
+      id: '0',
```

Comment:
Try to separate the variables, say `export const employees = []`, `export const candidates = []`, so that I don't have to use the data like `dummyData.employee`